com.tercad.fingrom.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.tercad.fingrom</id>
+  <name>Fingrom</name>
+  <summary>Open-source platform-agnostic financial accounting application</summary>
```

Comment:
please don't lie in the summary. `CC-BY-NC-ND-4.0` is NOT open source. This package might even be breaking the license since it it a derivative work.

Reply:
nonwithstanding that Creative Commons licenses are not code licenses.

Reply:
Someone except the author is not eligible to distribute. What's wrong with that?

Reply:
`Creative Commons Licenses: by making your data publicly available, you retain ownership of your objects`

Reply:
Surely, it's not a free license, content used "with author permission".

> That makes it source available not open source.

Ok, let me rephrase then, if you're arguing to "redistributed" as inevitable part of the project.

> And both the things you said about the CC-BY-NC-ND-4.0 license are wrong....

What exactly?

CC-BY-NC-ND or Creative Commons Attribution NonCommercial NoDerivs. With this license, the user (while attributing the original creator) can only share the work but not change it in any way or ever use it commercially.
This is technically still considered open since it does not have a commercial charge for sharing the work, but it does not offer any flexibility with the material should someone wish to remix and adapt it.

Reply:
> Open source = sources of the project are freely accessible. No contradictions

That makes it source available not open source.

And both the things you said about the CC-BY-NC-ND-4.0 license are wrong.

> Someone except the author is not eligible to distribute, or forking without pushing back the made change.

Reply:
In fact it will show up as proprietary in software stores/website as AppStream.license_is_free_license returns false.

<details> 
<summary> Image </summary>

![5b8344f1b6](https://github.com/flathub/flathub/assets/62639087/de36d9e5-b9e1-4fcd-a0fa-efb928ecbc67)

</details>

Reply:
> Ok, let me rephrase then, if you're arguing to "redistributed modified versions" as inevitable part of the open-source project.

Yes that is an essential part to qualify for open source, see the definition anywhere. You also should be using a OSI/FSF approved license.

Creative Commons licenses are explicitly not software license https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

If you want to use a license that restricts those freedoms, feel free but call it "Source code is available", not "open source". That term is only applied when certain conditions like above are met.

Reply:
I do understand that it's not recommended; but not restricted. Possibly, I've missed the one with defined conditions as for CC-BY-NC-ND-4.0. Furthermore, that project is a part of the book, and the book is a part of that project - https://www.amazon.com/Zero-Market-Flutter-Desktop-Distribution-ebook/dp/B0CLLKNXQR . They affect each other and use the single repository.